src/components/common/FileUploadComponent.tsx

```diff
@@ -72,7 +72,7 @@ export function FileUploadComponent(props: {
   return (
     <>
       {amountFilesInfo}
-      <List component="ul" className={classes.questionariesList}>
+      <List component="ul" className={classes.questionnairesList}>
```

Comment:
Maybe we should stick with "questionaries" for consistency? I'm no language expert but when I looked into it a long time ago both forms seem to be acceptable synonyms "questionnaire" stemming from French, while "questionary" would be a standard English form. Feel free to correct me if I'm wrong :)

Reply:
Well yeah, it is an annoying word, but I wanted to change it to make spell checkers happy (vscode spell checker, Grammarly, Chrome complains about it, probably for historical reasons the 'questionaries' is also acceptable it may just not used in modern texts, just like you use 'your' and not 'thy'), and it also makes it easier to catch misspellings.

For the sake of consistency, we can also go through everything and rename it.